CONTRIBUTING.md

```diff
@@ -92,35 +111,55 @@ root@vcluster-0:/vcluster#
-```
-This should deploy the hostpath-mapper as a deployment instead of a daemonset and take care of other modifications to be made and should allow you to develop the hostpath-mapper itself against the syncer.
+### Debug vcluster with Delve
 
-#### Debug vcluster with Delve
-If you wish to run vcluster in the debug mode with delve, run `devspace dev -n vcluster` and wait until you see the command prompt (`root@vcluster-0:/vcluster#`).  
-Run `dlv debug ./cmd/vcluster/main.go --listen=0.0.0.0:2345 --api-version=2 --output /tmp/__debug_bin --headless --build-flags="-mod=vendor" -- start`  
-Wait until the `API server listening at: [::]:2345` message appears.  
-Start the `Debug vcluster (localhost:2346)` configuration in VSCode to connect your debugger session.  
-If you are not using VSCode, configure your IDE to connect to `localhost:2346` for the "remote" delve debugging.  
-**Note:** vcluster won't start until you connect with the debugger.  
-**Note:** vcluster will be stopped once you detach your debugger session.  
+If you wish to run vcluster in the debug mode with delve, run `devspace dev -n vcluster` and wait until you see the command prompt (`root@vcluster-0:/vcluster#`).
+Run `dlv debug ./cmd/vcluster/main.go --listen=0.0.0.0:2345 --api-version=2 --output /tmp/__debug_bin --headless --build-flags="-mod=vendor" -- start`
+Wait until the `API server listening at: [::]:2345` message appears.
+Start the `Debug vcluster (localhost:2346)` configuration in VSCode to connect your debugger session.
+If you are not using VSCode, configure your IDE to connect to `localhost:2346` for the "remote" delve debugging.
+**Note:** vcluster won't start until you connect with the debugger.
+**Note:** vcluster will be stopped once you detach your debugger session.
 
 ### Running vcluster Tests
 
-You can run the unit test suite with `./hack/test.sh` which should execute all the vcluster tests.  
+You can run the unit test suite with `./hack/test.sh` which should execute all the vcluster tests.
 
-The e2e test suite can be run from the e2e folder(`cd e2e`) with this command - `go test -v -ginkgo.v`.  
+The e2e test suite can be run from the e2e folder(`cd e2e`) with this command - `go test -v -ginkgo.v`.
 Alternatively, if you [install ginkgo binary](https://github.com/onsi/ginkgo#global-installation), you can run it with `ginkgo -v`.
 
-For running conformance tests, please take a look at [conformance tests](https://github.com/loft-sh/vcluster/tree/main/conformance/v1.21)
+For running conformance tests, please take a look at [conformance tests](https://github.com/loft-sh/tree/vcluster/main/conformance/v1.21)
+
+## Build vcluster CLI tool
+
+Build:
+
+```
+go generate ./... && go build -o vcluster cmd/vclusterctl/main.go # build vcluster cli
+./vcluster create v1 # create vcluster
+```
+
+Run:
+
+```
+go generate ./... && go build -o vcluster cmd/vclusterctl/main.go # build vcluster cli
+./vcluster create v1 # create vcluster
+```
```

Comment:
Why is there run and build which are the exact same commands?

Reply:
fixed